mibs/controlbox/CONTROLBOX-TH-332-MIB

```diff
@@ -0,0 +1,109 @@
+-- CONTROLBOX SNMP STACK SMIV2 SPECIFICATION MIB
```

Comment:
Could you rename this file to `CONTROLBOX-TH332-MIB` ?

Reply:
File CONTROLBOX-TH-332-MIB renamed to CONTROLBOX-TH332-MIB